### ISSUE TITLE
Use ACE loss for English handwriting recognition，but not converge

### ISSUE THREAD

**Issue by liudatutu** [summerlvsong/Aggregation-Cross-Entropy#14]
I use the ACE loss function to do English handwriting recognition. When the model is trained, it does not converge, but with CTC, it gets a good convergence effect. How can this happen?

**Comment by yflv-yanxia** [comment]
setting a larger lr might work

**Comment by MiTeng0215** [comment]
> setting a larger lr might work

Regarding the CRNN + ACE model for 1D prediction, I did a lot of experiments, but I did not get good results. I would like to ask you some details about setting the parameter T. My QQ is: 852057734, I hope to get your reply. Thank you

**Comment by bjlgcxc** [comment]
I also find this problem in the training of crnn with synth90k dataset
